carbon-design-system/carbon-for-ibm-dotcom
# fix(card-in-card): adjust card in card wrapper width on Collapsed grid mode

### Related Ticket(s)

https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues/5485

### Description

- This PR fixes the grid alignment issue seen on the `Card Group with Card in Card` variant when the grid mode is changed to `collapsed`. 

### Changelog

- Just some CSS styling changes.

### Testing Instructions

- Visit the `Card Group with Card in Card` variant - `?path=/story/components-card-group--with-card-in-card`
- Adjust the `Grid Mode` knob to `Collapsed (1px)`
- Ensure that the card within the `Card in Card` component aligns with the Card Group items.
- Ensure that the border-bottom line is set correctly on all breakpoints.


**Current:**
![68747470733a2f2f696d616765732e7a656e68756275736572636f6e74656e742e636f6d2f3562653039633361363130323930306336643136643939312f38626461323332332d666433632d343734392d623137662d653434303437396332663166](https://user-images.githubusercontent.com/1815714/111184581-79793d80-8587-11eb-9370-f37b38530c3c.jpg)


**Expected:**
![Screen Shot 2021-03-15 at 12 39 11 PM](https://user-images.githubusercontent.com/1815714/111188488-7a13d300-858b-11eb-9194-fc36c11fd51d.png)
